### PR TITLE
Include battles in output of userauth command

### PR DESCRIPTION
**Pull request by urkerab**
Someone in some room or other suggested that the userauth command should also show any battles where the user has auth.

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -916,25 +916,25 @@ exports.commands = {
 		if (group) {
 			buffer.push('Global auth: ' + group.charAt(0));
 		}
-		for (let i = 0; i < Rooms.global.chatRooms.length; i++) {
-			let curRoom = Rooms.global.chatRooms[i];
+		for (let id in Rooms.rooms) {
```
Comment (Slayer95) [inline commands.js]: Not sure if this is worth it. Maybe restrict it to active battles the user has joined?

**Comment by TheImmortal** [comment]
What's the point of this since battles expire

**Comment by Zarel** [comment]
@urkerab, what's the use-case here?

**Comment by Hidden50** [comment]
I find that often times, when I get a report about a battle, I first have to explain to the user how they can get back into it. If the player rank showed on their /userauth, I could enter it directly instead.

**Comment by Zarel** [comment]
The player rank doesn't go away when the battle ends, though, so the user's rank list would be very cluttered...

This could be useful, though.

File: commands.js (hunk 1 above)
Comment (Zarel) [inline commands.js]: In hindsight, this is an outer loop so it's not a very big deal.

**Comment by Hidden50** [comment]
> The player rank doesn't go away when the battle ends, though, so the user's rank list would be very cluttered...

Is it removed when the battle expires? If not, that should probably be the case.

**Comment by Zarel** [comment]
The room itself is removed when the battle expires, so the room rank, being part of the room, would of course also no longer exist.

**Pull request merged**